test/js/apply-preserving-inline-style.js

```diff
@@ -69,4 +75,114 @@ suite('apply-preserving-inline-style', function() {
     this.style.cssText = 'top: 0px';
     assert.equal(this.style.length, 1);
   });
+  test('Detect SVG transform compatibility', function() {
+    var win = {navigator: {userAgent: ''}};
```

Comment:
It's a bit strange to have this local variable, agent() can just return an object.